src/react/hooks/use-messages.ts

```diff
@@ -101,14 +107,19 @@ export const useMessages = (params?: UseMessagesParams): UseMessagesResponse =>
     [context],
   );
   const deleteMessage = useCallback(
-    (message: Message, deleteMessageParams?: DeleteMessageParams) =>
+    (message: Message, deleteMessageParams?: ActionDetails) =>
```

Comment:
Ahh so sorry, I missed this! It should be `DeleteMessageParams` here

Reply:
done